interfaces/HO-Portal/src/app/program/program-people/program-people.component.ts

```diff
@@ -200,38 +328,63 @@ export class ProgramPeopleComponent implements OnChanges {
           pa: `PA #${index + 1}`,
           score: person.score,
           did: person.did,
-          created: formatDate(person.created, this.dateFormat, this.locale),
-          updated: formatDate(person.updated, this.dateFormat, this.locale),
+          processStarted: formatDate(person.created, this.dateFormat, this.locale),
+          digitalIdCreated: person.appliedDate ? formatDate(person.appliedDate, this.dateFormat, this.locale) : null,
+          digitalIdValidated: person.validationDate ? formatDate(person.validationDate, this.dateFormat, this.locale) : null,
+          vulnerabilityAssessmentCreated: person.appliedDate ? formatDate(person.appliedDate, this.dateFormat, this.locale) : null,
+          vulnerabilityAssessmentValidated: person.validationDate ? formatDate(person.validationDate, this.dateFormat, this.locale) : null,
+          inclusionCommunication: person.inclusionDate ? formatDate(person.inclusionDate, this.dateFormat, this.locale) : null,
+          included: person.included ? 'Included' : (person.excluded ? 'Excluded' : ''),
         };
 
+        this.paymentColumns.map((_, index2) => {
+          const payment = pastInstallments.find(i => i.installment === index2 + 1);
+          if (payment) {
+            personData['payment' + (index2 + 1)] = formatDate(payment.installmentDate, this.dateFormat, this.locale);
+          }
+        });
+
         if (person.name) {
           personData.name = person.name;
         }
         if (person.dob) {
           personData.dob = person.dob;
         }
-        if (person.included) {
-          personData.included = person.included;
-        }
 
         return personData;
       });
   }
 
-  private defaultSelectedPeople(source: Person[]): Person[] {
-    if (this.program.inclusionCalculationType === InclusionCalculationType.highestScoresX) {
-      const nrToInclude = this.program.highestScoresX;
+  private defaultSelectedPeople(source: any[]): any[] {
+    if (this.selectedPhase === ProgramPhase.inclusion) {
+      if (this.program.inclusionCalculationType === InclusionCalculationType.highestScoresX) {
+        const nrToInclude = this.program.highestScoresX;
 
-      return source.slice(0, nrToInclude);
-    }
+        return source.slice(0, nrToInclude);
+      }
+
+      const minimumScore = this.program.minimumScore;
 
-    const minimumScore = this.program.minimumScore;
+      return source.filter((person) => person.score >= minimumScore);
+    } else {
+      return [];
+    }
+  }
 
-    return source.filter((person) => person.score >= minimumScore);
+  private defaultSelectedPeoplePrivacy(source: any[]): any[] {
+    if (
+      this.selectedPhase === this.activePhase &&
+      [ProgramPhase.inclusion, ProgramPhase.finalize, ProgramPhase.payment].includes(ProgramPhase[this.selectedPhase])
```

Comment:
I wonder if we can move all these types of logic into a more central place...

We should/could then create multiple methods (side-by-side) that only do 1 thing (for example: `filterOnlyIncluded()`).
And then 1 'controller'/'init'-method picks and chooses from these available methods.